### ISSUE TITLE
检测模型训练没有输出loss等日志信息

### ISSUE THREAD

**Issue by JawerZ** [PaddlePaddle/PaddleOCR#468]
`2020-08-03 12:21:39,396-INFO: {'Global': {'debug': False, 'algorithm': 'EAST', 'use_gpu': False, 'epoch_num': 100000, 'log_smooth_window': 20, 'print_batch_step': 5, 'save_model_dir': './output/det_east/', 'save_epoch_step': 10000, 'ev
al_batch_step': [5000, 5000], 'train_batch_size_per_card': 8, 'test_batch_size_per_card': 116, 'image_shape': [3, 512, 512], 'reader_yml': './configs/det/det_east_icdar15_reader.yml', 'pretrain_weights': './pretrain_models/ResNet50_vd_
ssld_pretrained/', 'save_res_path': './output/det_east/predicts_east.txt', 'checkpoints': './output/det_EAST/best_accuracy', 'save_inference_dir': None, 'use_space_char': True}, 'Architecture': {'function': 'ppocr.modeling.architecture
s.det_model,DetModel'}, 'Backbone': {'function': 'ppocr.modeling.backbones.det_resnet_vd,ResNet', 'layers': 50}, 'Head': {'function': 'ppocr.modeling.heads.det_east_head,EASTHead', 'model_name': 'large'}, 'Loss': {'function': 'ppocr.mo
deling.losses.det_east_loss,EASTLoss'}, 'Optimizer': {'function': 'ppocr.optimizer,AdamDecay', 'base_lr': 0.0001, 'beta1': 0.9, 'beta2': 0.999}, 'PostProcess': {'function': 'ppocr.postprocess.east_postprocess,EASTPostPocess', 'score_th
resh': 0.8, 'cover_thresh': 0.1, 'nms_thresh': 0.2}, 'TrainReader': {'reader_function': 'ppocr.data.det.dataset_traversal,TrainReader', 'process_function': 'ppocr.data.det.east_process,EASTProcessTrain', 'num_workers': 1, 'img_set_dir'
: './train_data/train/', 'label_file_path': './train_data/train_det_label.txt', 'background_ratio': 0.125, 'min_crop_side_ratio': 0.1, 'min_text_size': 1}, 'EvalReader': {'reader_function': 'ppocr.data.det.dataset_traversal,EvalTestRea
der', 'process_function': 'ppocr.data.det.east_process,EASTProcessTest', 'img_set_dir': './train_data/test/', 'label_file_path': './train_data/test_det_label.txt'}, 'TestReader': {'reader_function': 'ppocr.data.det.dataset_traversal,Ev
alTestReader', 'process_function': 'ppocr.data.det.east_process,EASTProcessTest', 'infer_img': None, 'img_set_dir': './train_data/test/', 'label_file_path': './train_data/test_det_label.txt', 'do_eval': True}}
2020-08-03 12:21:41,645-INFO: places would be ommited when DataLoader is not iterable
2020-08-03 12:21:44,960-INFO: Finish initing model from ./output/det_EAST/best_accuracy
2020-08-03 12:21:44,960-INFO: During the training process, after the 5000th iteration, an evaluation is run every 5000 iterations
W0803 12:21:45.811342  1876 build_strategy.cc:170] fusion_group is not enabled for Windows/MacOS now, and only effective when running with CUDA GPU.
W0803 12:21:46.173436  1876 fuse_all_reduce_op_pass.cc:74] Find all_reduce operators: 198. To make the speed faster, some all_reduce ops are fused during training, after fusion, the number of all_reduce ops is 198.
2020-08-03 12:45:34,097-INFO: Already save model in ./output/det_east//iter_epoch_10000
2020-08-03 13:04:33,037-INFO: Already save model in ./output/det_east//iter_epoch_20000
2020-08-03 13:23:33,422-INFO: Already save model in ./output/det_east//iter_epoch_30000
2020-08-03 13:42:35,780-INFO: Already save model in ./output/det_east//iter_epoch_40000
2020-08-03 14:01:48,962-INFO: Already save model in ./output/det_east//iter_epoch_50000
`
只保存了model 但是model进行测试无法检测到文本

**Comment by MissPenguin** [comment]
没有正常训练，是只替换了训练数据吗，建议先按照教程示例确认能正常启动训练，然后一步步替换数据和pretrain模型，方便debug。另外，ev al_batch_step，Ev alTestReader 中间怎么都多了一个空格。。

**Comment by JawerZ** [comment]
![image](https://user-images.githubusercontent.com/59463796/89359569-be3b1d00-d6f8-11ea-9d19-748a86d46fb4.png)
按照教程会有这样的问题
请问是什么原因

> 没有正常训练，是只替换了训练数据吗，建议先按照教程示例确认能正常启动训练，然后一步步替换数据和pretrain模型，方便debug。另外，ev al_batch_step，Ev alTestReader 中间怎么都多了一个空格。。

**Comment by JawerZ** [comment]
> 没有正常训练，是只替换了训练数据吗，建议先按照教程示例确认能正常启动训练，然后一步步替换数据和pretrain模型，方便debug。另外，ev al_batch_step，Ev alTestReader 中间怎么都多了一个空格。。
我自己训练的时候只是替换了数据和预训练模型
空格是log复制过来的原因

**Comment by MissPenguin** [comment]
> ![image](https://user-images.githubusercontent.com/59463796/89359569-be3b1d00-d6f8-11ea-9d19-748a86d46fb4.png)
> 按照教程会有这样的问题
> 请问是什么原因
> 
打印log格式化出错，试试把train.py的113行改成这样：
                logger.info("reader: {}".format(count, len(data), batch_time))

**Comment by JawerZ** [comment]
> > ![image](https://user-images.githubusercontent.com/59463796/89359569-be3b1d00-d6f8-11ea-9d19-748a86d46fb4.png)
> > 按照教程会有这样的问题
> > 请问是什么原因
> 
> 打印log格式化出错，试试把train.py的113行改成这样：
> logger.info("reader: {}".format(count, len(data), batch_time))


![image](https://user-images.githubusercontent.com/59463796/89383503-2c023b80-d72f-11ea-855f-df3088c4c04a.png)
好像读不到数据

**Comment by MissPenguin** [comment]
方便提供你的训练数据吗，给你看下

**Comment by JawerZ** [comment]
> 方便提供你的训练数据吗，给你看下

可以的 怎么给你

**Comment by MissPenguin** [comment]
提供百度云下载链接，或者打包发到 601747729@qq.com吧

**Comment by JawerZ** [comment]
> 提供百度云下载链接，或者打包发到 [601747729@qq.com](mailto:601747729@qq.com)吧

已经发邮箱了 感谢帮看！

**Comment by MissPenguin** [comment]
> > 提供百度云下载链接，或者打包发到 [601747729@qq.com](mailto:601747729@qq.com)吧
> 
> 已经发邮箱了 感谢帮看！

我这边直接替换数据为你提供的数据，是可以正常训练的呢
![image](https://user-images.githubusercontent.com/25809855/90097781-5b392e00-dd69-11ea-8cb1-59258bdf9255.png)

只改动了这里：
![image](https://user-images.githubusercontent.com/25809855/90097820-6e4bfe00-dd69-11ea-96c9-718491b66891.png)

**Comment by MissPenguin** [comment]
上面是r50vd east的情况，下面是mv3 db的训练情况，也是正常的：
![image](https://user-images.githubusercontent.com/25809855/90098232-5a54cc00-dd6a-11ea-8deb-822d0d5a5d98.png)

**Comment by MissPenguin** [comment]
先确保原始环境的任务能跑通，再进行改动，这样比较好debug哈

**Comment by JawerZ** [comment]
> 先确保原始环境的任务能跑通，再进行改动，这样比较好debug哈

好的  谢谢

**Comment by caopulan** [comment]
请问问题是怎么解决的，我按照文档进行训练也遇到了这个问题
